### PR TITLE
Fix configuration: ShortNames

### PR DESCRIPTION
**Pull request by rosenrodt**
Depends on #1242

- Fix global parameter "ShortNames"; brought to you by @aazz44ss
- Remove `xfail` status of TensileCreateLibrary integration test involving the said parameter

### CODE REVIEW

**Review by aazz44ss — APPROVED** [review]
Old:
"shortNames" will modify file name and kernel name to SerialName.
it will has failure when doing TensileCreateLibrary.
and also hard for developer to search kernel.

Now:
"shortNames" will only modify file name to SerialName.

File: Tensile/KernelWriter.py [hunk 1]
```diff
@@ -3596,7 +3592,10 @@ def getReplacementKernelPath(self, kernel):
     kernelName = self.getKernelName(kernel)
     return ReplacementKernels.Get(kernelName)
 
-  def shortenFileBase(self, base):
+  def shortenFileBase(self, kernel):
```
Comment (sdquiring) [inline Tensile/KernelWriter.py:3595]: This function's name no longer makes sense.  It should still just shorten the input.  The serial name should be implemented in `getKernelFileBase()` or in a function called from there.

**Review by sdquiring — CHANGES_REQUESTED** [review]

File: Tensile/KernelWriter.py (hunk 1 above)
Comment (aazz44ss) [reply Tensile/KernelWriter.py:3599]: move ShortNames to getKernelFileBase()

**Review by sdquiring — APPROVED** [review]

**Review by cgmillette — APPROVED** [review]
Looks good to me

**Pull request merged**